contracts/acl/ACL.sol

```diff
@@ -320,6 +320,34 @@ contract ACL is IACL, AragonApp, ACLHelpers {
         return false;
     }
 
+    function checkOracle(address _oracleAddr, address _who, address _where, bytes32 _what, uint256[] _how) internal view returns (bool) {
+        bytes4 sig = ACLOracle(_oracleAddr).canPerform.selector;
+
+        // a raw call is required so we can return false if the call reverts, rather than reverting
+        bool ok = _oracleAddr.call(sig, bytes32(_who), bytes32(_where), _what, 0x80, _how.length, _how);
+        // 0x80 is the position where the array that goes there starts
+
+        if (!ok) {
+            return false;
+        }
+
+        uint256 size;
+        assembly { size := returndatasize }
+        if (size != 32) {
+            return false;
+        }
+
+        bool result;
+        assembly {
+            let ptr := mload(0x40)
+            returndatacopy(ptr, 0, size)
+            result := mload(ptr)
+            mstore(ptr, 0)
```

Comment:
Might be useful to add a few comments here to explain exactly what we're doing (copying the return into memory, loading the memory, and then clearing the memory).